### PR TITLE
[Hotfix][Docs]Correct errors in the FAQ doc

### PR DESCRIPTION
**Pull request by davidzollo**
There are a lot of old-time FAQs on the official website. I just closed the last PR and reorganized these questions, thanks for your review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Yes, SeaTunnel is available under the Apache 2.0 License, allowing commercial us
 
 Our [Official Documentation](https://seatunnel.apache.org/docs) includes detailed guides and tutorials to help you get started.
 
-### 7. Is there a community or support channel?
+### 6. Is there a community or support channel?
 
 Join our Slack community for support and discussions: [SeaTunnel Slack](https://s.apache.org/seatunnel-slack).
+more information, please refer to [FAQ](https://seatunnel.apache.org/docs/faq).
```
Comment (Hisoka-X) [inline README.md]:
```suggestion
Join our Slack community for support and discussions: [SeaTunnel Slack](https://s.apache.org/seatunnel-slack).
More information, please refer to [FAQ](https://seatunnel.apache.org/docs/faq). 
```

File: docs/en/faq.md [hunk 2]
```diff
@@ -1,332 +1,123 @@
...
-
-You can use the parameter `-i` or `--variable` followed by `key=value` to specify the value of the variable, where the key needs to be same as the variable name in the configuration.
-
-## How do I write a configuration item in multi-line text in the configuration file?
+## Does SeaTunnel support CDC from MySQL replicas? How are logs pulled?
+Yes, SeaTunnel supports CDC from MySQL replicas by subscribing to binlog logs, which are then parsed on the SeaTunnel server.
 
-When a configured text is very long and you want to wrap it, you can use three double quotes to indicate its start and end:
+## Does SeaTunnel support CDC integration for tables without primary keys?
+No, SeaTunnel does not support CDC integration for tables without primary keys. This is because, in cases where two identical records exist in the upstream and one is deleted or modified, the downstream cannot determine which record to delete or modify, leading to potential issues. Having primary keys is essential for ensuring data uniqueness, similar to identifying the real Monkey King in the classic "Journey to the West."
```
Comment (Hisoka-X) [inline docs/en/faq.md]:
```suggestion
SeaTunnel does not support CDC integration for tables without primary keys. The reason is that if two identical records exist in the upstream and one is deleted or modified, the downstream cannot determine which record to delete or modify, leading to potential issues. Primary keys are essential to ensure data uniqueness.
```

File: docs/en/faq.md [hunk 3]
```diff
@@ -1,332 +1,123 @@
...
-## How do I implement variable substitution for multi-line text?
-
-It is a little troublesome to do variable substitution in multi-line text, because the variable cannot be included in three double quotation marks:
-
-```
-var = """
-your string 1
-"""${you_var}""" your string 2"""
-```
+## How does SeaTunnel handle changes in data sources (source) or data destinations (sink)?
```
Comment (Hisoka-X) [inline docs/en/faq.md]: What's `changes` meaning?

File: docs/en/faq.md [hunk 4]
```diff
@@ -1,332 +1,123 @@
...
+For more details, refer to the answers on `schema_save_mode` and `data_save_mode` below.
 
-## Is SeaTunnel supported in Azkaban, Oozie, DolphinScheduler?
+## Does SeaTunnel support automatic table creation?
+Before starting an integration task, you can select different handling schemes for existing table structures on the target side, controlled via the `schema_save_mode` parameter. Available options include:
+- **`RECREATE_SCHEMA`**: Creates the table if it does not exist; if the table exists, it is deleted and recreated.
+- **`CREATE_SCHEMA_WHEN_NOT_EXIST`**: Creates the table if it does not exist; skips creation if the table already exists.
+- **`ERROR_WHEN_SCHEMA_NOT_EXIST`**: Throws an error if the table does not exist.
+- **`IGNORE`**: Ignores table handling.
+  Many connectors currently support automatic table creation. Refer to the specific connector documentation, such as [Jdbc sink](https://seatunnel.apache.org/docs/2.3.8/connector-v2/sink/Jdbc#schema_save_mode-enum), for more information.
```
Comment (Hisoka-X) [inline docs/en/faq.md:41]:
```suggestion
  Many connectors currently support automatic table creation. Refer to the specific connector documentation, such as [Jdbc sink](https://seatunnel.apache.org/docs/connector-v2/sink/Jdbc#schema_save_mode-enum), for more information.
```

File: docs/en/faq.md [hunk 5]
```diff
@@ -1,332 +1,123 @@
...
-
-## When SeaTunnel developers develop their own plugins, do they need to understand the SeaTunnel code? Should these plugins be integrated into the SeaTunnel project?
-
-The plugin developed by the developer has nothing to do with the SeaTunnel project and does not need to include your plugin code.
+For more details, see: [lightbend/config#456](https://github.com/lightbend/config/issues/456).
 
-The plugin can be completely independent from SeaTunnel project, so you can write it using Java, Scala, Maven, sbt, Gradle, or whatever you want. This is also the way we recommend developers to develop plugins.
+## How do I configure logging parameters for SeaTunnel E2E Tests?
+The log4j configuration file for `seatunnel-e2e` is located at `seatunnel-e2e/seatunnel-e2e-common/src/test/resources/log4j2.properties`. You can directly modify logging-related parameters in this configuration file. For example, to produce more detailed E2E Test logs, lower the `rootLogger.level` in the configuration file.
```
Comment (Hisoka-X) [inline docs/en/faq.md]:
```suggestion
```
This question look likes useless for users.

**Comment by davidzollo** [comment]
@Hisoka-X Thanks for your suggestions, I have updated the en and ch docs

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**